flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/batch/table/CalcTest.scala

```diff
@@ -330,6 +330,57 @@ class CalcTest extends TableTestBase {
 
     util.verifyTable(resultTable, expected)
   }
+
+  @Test
+  def testAddColumns(): Unit = {
+    val util = batchTestUtil()
+    val sourceTable = util.addTable[(Int, Long, String)]("MyTable", 'a, 'b, 'c)
+    val resultTable = sourceTable
+      .addColumns("concat(c, 'sunny') as kid")
```

Comment:
Could we split that complex test into multiple tests that describe what is tested? Those tests do not spawn cluster, so they are cheap, we can have them as many as we like.

What I mean by splitting is having tests like `testSimpleAdd`, `testReplaceIfExist` etc. Right now it costs the reader a lot of effort to understand what is being checked.

Reply:
I think you are right. +1